### PR TITLE
Bring mosaic in dali

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**

### CODE REVIEW

**Review by cor — APPROVED** [review]
LGTM

File: runtime/dali/Cargo.toml [hunk 1]
```diff
@@ -172,7 +173,8 @@ std = [
   "crowdloan-rewards/std",
   "bonded-finance/std",
 	"dutch-auction/std",
-	"vesting/std"
+	"vesting/std",
+  "mosaic/std"
```
Comment (dzmitry-lahoda) [inline runtime/dali/Cargo.toml:177]: spacing

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by seunlanlege — APPROVED** [review]

**Comment by seunlanlege** [comment]
missing benchmarks for mosaic in dali 🙏🏿

**Comment by KaiserKarel** [comment]
> missing benchmarks for mosaic in dali 🙏🏿

For pallet-mosaic, or in general for multiple pallets?

**Comment by hussein-aitlahcen** [comment]
> > missing benchmarks for mosaic in dali 🙏🏿
> 
> For pallet-mosaic, or in general for multiple pallets?

Benchmarking is missing for a lot of pallets

**Pull request merged**